ecosystem/nft-metadata-crawler/nft-metadata-crawler-parser/src/parser_entry.rs

```diff
@@ -168,21 +174,67 @@ impl ParserEntry {
     }
 
     /// Calls and handles error for writing JSON to GCS
-    async fn handle_write_json_to_gcs(&mut self, _json: Option<Value>) -> Option<String> {
-        todo!();
+    async fn handle_write_json_to_gcs(&mut self, json: Option<Value>) -> Option<String> {
+        if let Some(json) = json {
+            match write_json_to_gcs(
+                self.token.clone(),
+                self.bucket.clone(),
+                self.entry.token_data_id.clone(),
+                json,
+            )
+            .await
+            {
+                Ok(filename) => {
+                    self.log_info("Successfully wrote JSON to GCS");
```

Comment:
btw why are you implementing your own log wrapper? Isn't tracing::info! enough? 

Another thing is that your logs are basically useless as is b/c there is no metadata. We need things like the token_data_id, bucket, etc. Please use structured logs as well, e.g. 

tracing::info!(token_data_id = self.entry.token_data_id, "blabla");